sqlglot/expressions.py

```diff
@@ -477,6 +501,39 @@ class Select(Expression):
         "offset": False,
     }
 
+    @classmethod
+    def from_(cls, selectable, selects=None):
+        selects = selects or (Star(),)
+        selects = [s if isinstance(s, Expression) else Column(this=s) for s in selects]
+
+        if isinstance(selectable, str):
+            selectable = Table(this=selectable)
+
+        return cls(**{"from": From(expressions=[selectable]), "expressions": selects})
+
+    @builder
+    def select(self, *selects):
+        selects = selects or (Star(),)
```

Comment:
I dont think selecting star here is a good fallback. Consider this example:
```python
query = ...
columns = []
if foo:
  columns.appned('foo')
query = query.select(columns) 
...
query = query.select('bar')
```

This now would be `select foo, bar` or `select *, bar`

Reply:
Removed all this defaulting.